### ISSUE TITLE
Docker doesn't have 2.4.4

### ISSUE THREAD

**Issue by ThinkChaos** [caddyserver/dist#72]
Hi, I'm not sure if this is the correct repo to report this, but v2.4.4 wasn't released on docker hub.

**Comment by mholt** [comment]
It always takes a few days, and we're planning on releasing a hotfix in 2.4.5 soon anyway.

**Comment by francislavoie** [comment]
The docker image is maintained in https://github.com/caddyserver/caddy-docker

We're delaying the Docker image release due to a somewhat critical bug we found after we released in v2.4.4, which will be fixed with v2.4.5. https://github.com/caddyserver/caddy/issues/4314

There's a bunch of manual steps involved in building and releasing new versions of the Docker image. We need to update https://github.com/caddyserver/caddy-docker with the build metadata, then make a pull request to https://github.com/docker-library/official-images for the official images maintainers to review, and then it gets queued up in their Jenkins build queue which can take up to a few hours after that to get tagged and pushed to Docker Hub.

So we might as well wait for the version with the fix.
